docs/user/troubleshooting/E1001.md

```diff
@@ -2,16 +2,16 @@
 
 ## Message emitted:
 
-`E1001`: Did not find a valid dispatcher in {checked_path}, preemptively failed.
+`E1001`: Did not find a valid dispatcher in `{checked_path}`, preemptively failed.
 
 ## Description:
 
-Coming soon....
+A dispatcher of `{checked_path}` was provided but not found.
 
 ## Troubleshooting:
 
-Coming soon....
+Ensure that the dispatcher path is correct and that the dispatcher is installed in both the web server and worker.
 
 ## Recommendation:
 
-Coming soon....
\ No newline at end of file
+Check that the dispatcher path is correctly spelled and that the dispatcher is installed in both the web server and worker.
```

Comment:
```suggestion
Check that the dispatcher path is correctly spelled and that the dispatcher is installed in both the web server and worker.
Manually go into `nautobot-server nbshell` and attempt the import manually.
```

Reply:
have to update constants instead, so I did that